engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/compile/ProcessValidatorSpec.scala

```diff
@@ -93,11 +93,31 @@ class ProcessValidatorSpec extends FunSuite with Matchers with Inside {
     ObjectDefinition.noParam,
     ExpressionDefinition(
       Map("processHelper" -> ObjectDefinition(List(), Typed(ProcessHelper.getClass), List("cat1"), SingleNodeConfig.zero)),
-      List.empty, LanguageConfiguration.default, optimizeCompilation = false, strictTypeChecking = true, dictionaries = Map.empty, hideMetaVariable = false, strictMethodsChecking = true
+      List.empty, LanguageConfiguration.default, optimizeCompilation = false, strictTypeChecking = true, dictionaries = Map.empty, hideMetaVariable = false, strictMethodsChecking = true, staticMethodInvocationsChecking = true
     ),
     ClassExtractionSettings.Default
   )
 
+  test("Invocation of Type Reference of valid class, should succeed") {
```

Comment:
1. Should be "Validation of Type Reference using accessible class, should succeed"
2. What about my comment "I think that we need to add more tests in SpelExpressionSpec that would check that not only epxression is valid but also would evaluate"?

Reply:
1. Changed
2. I added the tests in SpelExpressionSpec for combining validation with evaluation